docs/api/sandbox-option.md

```diff
@@ -56,35 +54,20 @@ only via IPC. The use of this option stops Electron from creating a Node.js runt
 within this new window `window.open` follows the native behaviour (by default Electron creates a [`BrowserWindow`](browser-window.md)
 and returns a proxy to this via `window.open`).
 
-It is important to note that this option alone won't enable the OS-enforced sandbox. To enable this feature, the
-`--enable-sandbox` command-line argument must be passed to electron, which will
-force `sandbox: true` for all `BrowserWindow` instances.
+[`app.enableSandbox`](app.md#appenablesandbox-experimental) can be used to force `sandbox: true` for all `BrowserWindow` instances.
 
 ```js
 let win
+app.enableSandbox()
 app.on('ready', () => {
-  // no need to pass `sandbox: true` since `--enable-sandbox` was enabled.
+  // no need to pass `sandbox: true` since `app.enableSandbox()` was called.
   win = new BrowserWindow()
   win.loadURL('http://google.com')
 })
 ```
 
-Note that it is not enough to call
-`app.commandLine.appendSwitch('--enable-sandbox')`, as electron/node startup
-code runs after it is possible to make changes to Chromium sandbox settings. The
-switch must be passed to Electron on the command-line:
-
-```sh
-electron --enable-sandbox app.js
-```
-
-It is not possible to have the OS sandbox active only for some renderers, if
-`--enable-sandbox` is enabled, normal Electron windows cannot be created.
-
-If you need to mix sandboxed and non-sandboxed renderers in one application,
-omit the `--enable-sandbox` argument. Without this argument, windows
-created with `sandbox: true` will still have Node.js disabled and communicate
-only via IPC, which by itself is already a gain from security POV.
+**Note:** It is not possible to have the OS sandbox active only for some renderers, if
```

Comment:
Found this note confusingly unnecessary as it seems to be saying "if you flip the switch to B, then you won't get A". If app.enableSandbox() is not called, then we do have the possibility of enabling the OS sandbox _only_ for the renderers of windows that you start with `sandbox: true`. It is the purpose of `enableSandbox()` to force that all renderers are OS sandboxed, so I don't see the need for this note and would remove it.

Reply:
removed